__init__.py

```diff
@@ -643,29 +648,38 @@ def _get_speakable_timer_details(self, timers: List[CountdownTimer]) -> str:
 
         return timer_names
 
-    def display_timers(self):
+    def _show_gui(self):
+        """Update the device's display to show the status of active timers.
+
+        Runs once a second via a repeating event to keep the information on the display
+        accurate.
+        """
+        if self.gui.connected:
+            self._update_gui()
+            if self.platform == MARK_II:
+                page = "timer_mark_ii.qml"
+            else:
+                page = "timer_scalable.qml"
+            self.gui.show_page(page, override_idle=True)
+
+    def update_display(self):
         """Update the device's display to show the status of active timers.
 
         Runs once a second via a repeating event to keep the information on the display
         accurate.
         """
         if self.gui.connected:
-            self._display_timers_on_gui()
+            self._update_gui()
         elif self.platform == MARK_I:
             self._display_timers_on_faceplate()
 
-    def _display_timers_on_gui(self):
+    def _update_gui(self):
         """Display active timers on a device that supports the QT GUI framework."""
         timers_to_display = self._select_timers_to_display(display_max=4)
         display_data = [timer.display_data for timer in timers_to_display]
-        if self.platform == MARK_II:
-            page = "timer_mark_ii.qml"
-        else:
-            page = "timer_scalable.qml"
         if timers_to_display:
             self.gui["activeTimers"] = dict(timers=display_data)
             self.gui["activeTimerCount"] = len(timers_to_display)
-            self.gui.show_page(page, override_idle=True)
 
     def _display_timers_on_faceplate(self):
```

Comment:
Where does faceplate_user get initialized?

Reply:
I am not sure I understand your question.  It is a local variable that gets assigned in the first statement of the `_display_timers_on_faceplate()` method.